Bank.md

```diff
@@ -1,37 +1,30 @@
-# Bank Configuration
```

Comment:
We have a very consistent format for plugin wiki pages, this shouldn't be removed (unless we're going to change all plugin wiki pages)

Reply:
Feels redundant to have the page title, and then right below it, "<page title> Configuration". I think it would make sense to remove this "<page title> Configuration> header on every page, what do you think?

Reply:
I wouldn't mind getting rid of the "plugin Configuration" header on every page, though idk if I like this change to remove (some of) the config headers.

In the long term, I'm still interested in incorporating plugin wiki page data into the client repo and building & pushing new pages with releases so that they will always be current as we update them. If that's going to be possible, there would still need to be a consistent structure like what exists currently. I'm not at all opposed to trimming the fat that exists currently though.